docs/content/docs/dev/table/sql/analyze.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "ANALYZE Statements"
+weight: 8
+type: docs
+aliases:
+  - /dev/table/sql/analyze.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# ANALYZE TABLE Statements
+
+{{< label Batch >}}
+
+- `ANALYZE TABLE` statements are used to collect statistics for existing tables, and write statistics back to catalog. 
+- Only existing table is supported, and an exception will be thrown if the table is a view or table not exists.
+- Currently, `ANALYZE TABLE` only supports in batch mode.
+- `ANALYZE TABLE` statements are triggered manually instead of automatically.
```

Comment:
ANALYZE statements are used to collect statistics for existing tables and store the result to catalog. Only `ANALYZE TABLE` statement is supported now, and need to be triggered manually instead of automatically.

`<span class="label label-danger">Attention</span> Currently, ANALYZE TABLE only supports in batch mode. Only existing table is supported, and an exception will be thrown if the table is a view or table not exists.`

Reply:
> ANALYZE statements are used to collect statistics for existing tables and store the result to catalog. Only `ANALYZE TABLE` statement is supported now, and need to be triggered manually instead of automatically.
> 
> `<span class="label label-danger">Attention</span> Currently, ANALYZE TABLE only supports in batch mode. Only existing table is supported, and an exception will be thrown if the table is a view or table not exists.`

Done!